original/pong/main.py

```diff
@@ -29,18 +29,24 @@
 )
 
 
+class Home(wa2.RequestHandler):
+    @util.require_login(Session)
+    def get(self):
+        return wa2.redirect('/users')
+
+
 class ViewGames(wa2.RequestHandler):
     @util.require_login(Session)
     def get(self):
         session = Session()
         games = session.query(models.Game).all()
         template = JINJA_ENVIRONMENT.get_template('view_games.html')
-        self.response.write(
-            template.render(
-                {'games': reversed(games),
-                 'logout_url': usersapi.create_logout_url('/games')}
-            )
-        )
+        content = template.render({
+            'games': reversed(games),
+            'logout_url': usersapi.create_logout_url('/games'),
+            'is_admin_user': usersapi.is_current_user_admin()
```

Comment:
I had originally added an "admin" link in the base template which would only render for admin users, so this would be needed for every template. It's not strictly needed right now, but I think an admin interface could be useful in the future, so I'm inclined to leave it.